flank-scripts/src/main/kotlin/flank/scripts/utils/Net.kt

```diff
@@ -0,0 +1,25 @@
+package flank.scripts.utils
+
+import java.net.InetAddress
+import java.net.InetSocketAddress
+import java.net.Socket
+
+internal fun isOnline(): Boolean {
+    val host = "1.1.1.1"
+
+    try {
+        val pingResult = InetAddress.getByName(host).isReachable(1000)
+        if (pingResult) return true
```

Comment:
maybe let's skip this useless `val`

`if (InetAddress.getByName(host).isReachable(1000)) return true`